lang/fr_FR.UTF-8/LC_MESSAGES/messages.po

```diff
@@ -16222,3 +16222,6 @@ msgstr "Options de recherche"
 
 msgid "The parameter \"%s\" must be an integer"
 msgstr "Le paramètre \"%s\" doit être un entier"
+
+msgid "Service graphs"
+msgstr "Graphiques de services"
```

Comment:
```suggestion
msgstr "Graphiques de service"
```

Reply:
I think it'd be more correect in french to talk about many services, in oppisition to english